cherryontech/jupiter-jumpers
# Adding styling updates to form and confirmation msg

## Description

- Updates styling for the registration form
- Updates styling for the confirmation modal
- Updates styling for the navbar

## Related Issue

[SCRUM-10 - Registration Form](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-10)
[SCRUM-12 - Confirmation message for registration](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-12)


## Related Story Card



## Type of Changes

Styling updates.

## Acceptance Criteria

N/A

## Update Screenshots


### Before
![Screenshot 2024-04-29 at 11 40 22 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/5a5c7964-4331-4763-9093-da5d202b7f7e)
![Screenshot 2024-04-29 at 11 40 44 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/dda2d0c0-b26c-4e8c-b85a-0bdf96510263)
![Screenshot 2024-04-29 at 11 41 09 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/b0998bec-a47d-4035-9f60-1a930ee8719d)

### After

<img width="1215" alt="Screenshot 2024-04-29 at 11 38 08 PM" src="https://github.com/cherryontech/jupiter-jumpers/assets/47403135/aa7db7ce-22ac-4f4b-ac6b-56c4bfc58828">
<img width="1215" alt="Screenshot 2024-04-29 at 11 38 58 PM" src="https://github.com/cherryontech/jupiter-jumpers/assets/47403135/95b0fa8d-a76b-4dad-a59a-5348ae034490">

<img width="1210" alt="Screenshot 2024-04-29 at 11 39 24 PM" src="https://github.com/cherryontech/jupiter-jumpers/assets/47403135/839f9ec2-b165-439b-b356-d288a65f1a8e">

## Testing Instructions

1. Try to create a new account

- Check the validation styling when you put in an incorrect email, name (longer than 20 chars), or password (shorter than 8 chars) (red border, red x; when you click on the red x it should clear the field).
- Check that the password toggle works (show/hide)
- Click 'Continue' and ensure you are redirected to the logged-in view

2. Check to make sure that the confirmation modal pops up after the initial account creation
- Press 'Get Started', and refresh the page to ensure that it does not pop back up

3. Press the logo in the top left corner, or the 'Unclouded' name; it should refresh to the main page. 

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
